docs/misc/changelog.rst

```diff
@@ -6,22 +6,27 @@ Changelog
 Pre-Release 0.11.0a0 (WIP)
 -------------------------------
 
-
 Breaking Changes:
 ^^^^^^^^^^^^^^^^^
+- ``VecFrameStack`` now uses ``channels_order`` argument to tell if observations should be stacked
```

Comment:
Is that really a breaking change as we are detecting the stack axis automatically?

Reply:
Some people (I know some) could be using FrameStackEnv on other-than-image-observations, like just on simple 1D observations, and with this update it will raise a an exception. Granted, this is not the case this stack-env is intended for but I can see why such use is tempting ^^